book/src/requirements.md

```diff
@@ -0,0 +1,102 @@
+# Requirements
+
+The design of Orchard aims to be mostly identical to the Sapling circuit in functionality.
+
+## Scope of Requirements
+
+These requirements are scoped to the protocol design itself, which excludes important broader considerations for Orchard as a whole, such as usability goals for a wallet which aren't directly constrained by the protocol. Additionally, an ideal criteria for these requirements is that any outside reviewer with sufficient domain expertise should be able to verify that the existing protocol specification meets the requirements in this document. This excludes requirements based on specific authorities / orgs / people.
+
+Finally, there is a category of proposed requirements that could have been coherently in-scope for Orchard which were excluded due to time & complexity constraints.
+
+These scope clarifications exlude some draft requirements found in earlier revisions of this document, in pull request comments, or elsewhere. Many of those could become ZIPs in their own right.
+
+## Protocol Product Requirements
+
+### ProtoPR1. Orchard Relies on a Zeroknowledge Proving System which is Secure in The Uniform Random String Model
+
+**Requirement:** The Orchard shielded transfer circuit uses a Zeroknowledge Proving System which is secure in the Uniform Random String (URS) Model, and therefore has no reliance on a trusted setup.
+
+**Rationale - UX:** Users who rely on Orchard are not directly vulnerable to the class of trusted parameter compromises.
+
+**Rationale - Strategic:** As Sprout and Sapling usage declines or are deprecated, the ZEC a whole will become protected from any previous trusted setup compromise henceforth.
+
+**Rationale - Strategic:** By avoiding a requirement on a trusted setup, Orchard *and future* upgrades no longer need a parameter setup MPC process, which improves protocol agility.
+
+### ProtoPR2. Orchard Addresses
+
+**Requirement:** Orchard introduces a new address format with an unambiguous user-facing encoding.
+
+**Rationale - UX:** A new address format allows users to identify which ZKP platform their funds rely on.
```

Comment:
Is this consistent with the proposal to switch to mainly relying on address bundles?

Reply:
Great point. The address bundle may be ambiguous depending on some trade-offs.

For example if it's intended to be backwards compatible with t-addr wallets and conform to [ZIP-321](https://zips.z.cash/zip-0321) then maybe it will be ambiguous to users.

In fact, the word "unambiguous" is ambiguous: does it mean "users don't confuse Orchard addresses for other kinds of addresses" or does it mean "the user facing encoding of Orchard addresses is unambiguously specified (so that all conforming implementations use the same encoding)". When I wrote this, I meant the former.

I'm going to leave this as a pending comment and circle around with people on Address Bundle clarity.